target/linux/ramips/image/common-sercomm.mk

```diff
@@ -22,14 +22,12 @@ define Build/sercomm-factory-cqr
 		--hw-version $(SERCOMM_HWVER) \
 		--hw-id $(SERCOMM_HWID) \
 		--sw-version $(SERCOMM_SWVER) \
-		--pid-file $@.fhdr
+		--pid-file $@.fhdr \
+		--extra-padding-size 0x190
```

Comment:
Please explain in the commit message why you add the extra padding here.

Reply:
Done. I added this to the commit message:
```
dd automatically fills the file by 0x00 if the filesize is less than
offset where we start writing. We drop such dd command so we need to add
--extra-padding-size 0x190 to the sercomm-pid.py call.
```

Reply:
And of course I checked SmartBox Turbo+ factory image. Before and after this commit. There is no difference.